### ISSUE TITLE
Cognitive_planning run train_supervised_active_vision.py show: cudaGetDevice() failed.

### ISSUE THREAD

**Issue by jsdd25** [tensorflow/models#5916]
Hi!
I have encountered some problems when I run train_supervised_active_vision.py in cognitive_planning. Please give me some advice. Thank you very much!

------------------------

### System information
- **What is the top-level directory of the model you are using**: cognitive_planning
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: 16.04
- **TensorFlow installed from (source or binary)**: anaconda
- **TensorFlow version (use command below)**: 1.12.0
- **Bazel version (if compiling from source)**: No
- **CUDA/cuDNN version**: 8.0.61
- **GPU model and memory**: GTX1080 8112MiB
- ** Exact command to reproduce**: 
```
~/cognitive_planning$ python train_supervised_active_vision.py \
>   --mode='train' \
>   --logdir=$CHECKPOINT_DIR \
>   --modality_types='det' \
>   --batch_size=8 \
>   --train_iters=200000 \
>   --lstm_cell_size=2048 \
>   --policy_fc_size=2048 \
>   --sequence_length=20 \
>   --max_eval_episode_length=100 \
>   --test_iters=194 \
>   --gin_config=envs/configs/active_vision_config.gin \
>   --logtostderr
```

### Describe the problem
```
(gymlab) jin@jin-System-Product-Name:~/cognitive_planning$ python train_supervised_active_vision.py \
>   --mode='train' \
>   --logdir=$CHECKPOINT_DIR \
>   --modality_types='det' \
>   --batch_size=8 \
>   --train_iters=200000 \
>   --lstm_cell_size=2048 \
>   --policy_fc_size=2048 \
>   --sequence_length=20 \
>   --max_eval_episode_length=100 \
>   --test_iters=194 \
>   --gin_config=envs/configs/active_vision_config.gin \
>   --logtostderr
I1217 09:06:55.325814 139683146712832 train_supervised_active_vision.py:377] modality types: [<ModalityTypes.OBJECT_DETECTION: 2>]
W1217 09:06:55.326095 139683146712832 active_vision_dataset_env.py:389] environment does not terminate the episode if the agent is too close to the environment
I1217 09:06:55.484458 139683146712832 active_vision_dataset_env.py:253] loading targets: /home/jin/cognitive_planning/AVD_Minimal/Meta/annotated_targets.npy
I1217 09:06:55.494143 139683146712832 active_vision_dataset_env.py:258] loading depth: /home/jin/cognitive_planning/AVD_Minimal/Meta/depth_imgs.npy
I1217 09:07:09.453788 139683146712832 active_vision_dataset_env.py:262] processing depth
I1217 09:07:10.269463 139683146712832 active_vision_dataset_env.py:277] loading sseg: /home/jin/cognitive_planning/AVD_Minimal/Meta/sseg_crf.npy
I1217 09:07:14.758609 139683146712832 active_vision_dataset_env.py:281] processing sseg
I1217 09:07:15.061821 139683146712832 active_vision_dataset_env.py:293] loading imgs: /home/jin/cognitive_planning/AVD_Minimal/Meta/imgs.npy
I1217 09:07:26.205753 139683146712832 active_vision_dataset_env.py:302] logging done in 30.721297 seconds
/home/jin/gym/gym/logger.py:30: UserWarning: WARN: gym.spaces.Box autodetected dtype as <type 'numpy.uint8'>. Please provide explicit dtype.
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
/home/jin/gym/gym/logger.py:30: UserWarning: WARN: gym.spaces.Box autodetected dtype as <type 'numpy.float32'>. Please provide explicit dtype.
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
OrderedDict([(<ModalityTypes.OBJECT_DETECTION: 2>, (tf.float32, [20, 64, 64, 90])), (<ModalityTypes.GOAL: 4>, (tf.float32, [20, 5])), (<ModalityTypes.PREV_ACTION: 5>, (tf.float32, [20, 8]))])
300
300

W1217 09:07:27.327918 139683146712832 tf_logging.py:125] From /home/jin/cognitive_planning/policies.py:150: __init__ (from tensorflow.python.ops.rnn_cell_impl) is deprecated and will be removed in a future version.
Instructions for updating:
This class is deprecated, please use tf.nn.rnn_cell.LSTMCell, which supports all the feature this cell currently has. Please replace the existing code with tf.nn.rnn_cell.LSTMCell(name='basic_lstm_cell').

W1217 09:07:27.809431 139683146712832 tf_logging.py:125] From /home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/contrib/slim/python/slim/learning.py:737: __init__ (from tensorflow.python.training.supervisor) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.MonitoredTrainingSession
2018-12-17 09:07:27.963019: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-12-17 09:07:28.064821: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:964] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-12-17 09:07:28.065167: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8475
pciBusID: 0000:01:00.0
totalMemory: 7.92GiB freeMemory: 7.60GiB
2018-12-17 09:07:28.065181: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
I1217 09:07:28.065444 139683146712832 tf_logging.py:115] Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InternalError'>, cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version
Traceback (most recent call last):
  File "train_supervised_active_vision.py", line 503, in <module>
    app.run(main)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "train_supervised_active_vision.py", line 497, in main
    train()
  File "train_supervised_active_vision.py", line 490, in train
    session_config=tf.ConfigProto(allow_soft_placement=True),
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/contrib/slim/python/slim/learning.py", line 748, in train
    master, start_standard_services=False, config=session_config) as sess:
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/supervisor.py", line 1004, in managed_session
    self.stop(close_summary_writer=close_summary_writer)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/supervisor.py", line 832, in stop
    ignore_live_threads=ignore_live_threads)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/coordinator.py", line 389, in join
    six.reraise(*self._exc_info_to_raise)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/supervisor.py", line 993, in managed_session
    start_standard_services=start_standard_services)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/supervisor.py", line 730, in prepare_or_wait_for_session
    init_feed_dict=self._init_feed_dict, init_fn=self._init_fn)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/session_manager.py", line 288, in prepare_session
    config=config)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/training/session_manager.py", line 185, in _restore_checkpoint
    sess = session.Session(self._target, graph=self._graph, config=config)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/client/session.py", line 1551, in __init__
    super(Session, self).__init__(target, graph, config=config)
  File "/home/jin/anaconda2/envs/gymlab/lib/python2.7/site-packages/tensorflow/python/client/session.py", line 676, in __init__
    self._session = tf_session.TF_NewSessionRef(self._graph._c_graph, opts)
tensorflow.python.framework.errors_impl.InternalError: cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version
```
Then I check the version of my CUDA and nvidia driver version.
```
jin@jin-System-Product-Name:~$ nvidia-smi
Mon Dec 17 09:14:27 2018       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 384.130                Driver Version: 384.130
```
```
jin@jin-System-Product-Name:~$ cat /usr/local/cuda/version.txt
CUDA Version 8.0.61
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by meettyj** [comment]
same question.

**Comment by jsdd25** [comment]
@meettyj Can I get in touch with you in order to fix problems together? My email is 773903267@qq.com

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
